distribution/docs/src/main/resources/_contents/_data/location-attributes-table-contents.adoc

```diff
@@ -10,27 +10,27 @@
 
 |location.altitude-meters
 |Altitude of the resource in meters
-|Double
+|List of doubles
 |> 0
 | 
  
 |location.country-code
 |One or more country codes associated with the resource
-|String
-|http://www.iso.org/iso/country_codes[ISO_3166-1] alpha-3
+|List of Strings
+|< 1024 characters per entry; http://www.iso.org/iso/country_codes[ISO_3166-1] alpha-3
 codes
 | 
 
 |location.crs-code
 |Coordinate reference system code of the resource
-|String
-|< 1024 characters
+|List of Strings
+|< 1024 characters per entry
```

Comment:
There are thousands of CRS codes.  Not sure enumerating all of them is reasonable (at least here).